gnoi_cert/gnoi_cert.go

```diff
@@ -60,30 +63,25 @@ var (
 func main() {
 	flag.Parse()
 
-	if *ca == "" || *key == "" {
-		log.Exit("-ca and -key must be set with file locations")
-	}
 	if *targetCN == "" {
 		log.Exit("Must set a Common Name ID with -target_name.")
 	}
-
-	var err error
-	if caEnt, err = entity.FromFile(*ca, *key); err != nil {
```

Comment:
You can add a function to cretdentials to return an entity if already created here:
https://github.com/google/gnxi/blob/master/utils/credentials/credentials.go#L89

this function would fail if CA or CAKey are not present